webapp-django/crashstats/crashstats/templatetags/jinja_helpers.py

```diff
@@ -244,8 +244,8 @@ def bugzilla_submit_url(report, bug_product):
         'product': bug_product,
         'op_sys': op_sys,
         'rep_platform': report['cpu_name'],
-        'cf_crash_signature': '[@ {}]'.format(report['signature']),
-        'short_desc': 'Crash in {}'.format(report['signature']),
+        'cf_crash_signature': u'[@ {}]'.format(report['signature']),
+        'short_desc': u'Crash in {}'.format(report['signature']),
```

Comment:
Ah! I think I know what you mean now. Sorry, I think I get it now. 
You mean this scenario:
```python
>>> u'{}'.format('unicode sträng')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 11: ordinal not in range(128)
```

What my fix did, was to fix this error:
```python
>>> '{}'.format(u'unicode sträng')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
UnicodeEncodeError: 'ascii' codec can't encode character u'\xe4' in position 11: ordinal not in range(128)
```

So it's two different problems. Do you mind if I solve both of them using this PR? Or do you prefer a fresh new bug plus PR?